docs/getting-started.md

```diff
@@ -168,10 +168,10 @@ account, for developing your own study:
 1. Use the **Owner** drop-down menu, and select the account you want to own the
    repository (typically under your own account)
    ![Entering a name, owner and description, when creating a repository from the research template.](images/create-repository-owner-name.png)
-1. Type a name for your repository, and an optional description.
-1. Choose a repository visibility.  This would usually be "Public".
+1. Type a name for your repository, and an optional description
+1. Choose a [repository visibility](repositories.md#repository-access). This would usually be "Public"
    ![Entering a description and choosing to make a repository public or private, when creating a repository from the research template.](images/create-repository-public-private.png)
-1. There is an "Include all branches" option: it can be left disabled.
+1. There is an "Include all branches" option: this does **not** need to be selected.
```

Comment:
Are there any times when any external person would need to use it?

Reply:
This option is referring to the cloning of the opensafely/research-template repository as part of the setup and they only need to copy over the main branch in that case. It doesn't affect researchers creating branches later on.